.gitlab/builds.gitlab-ci.yml

```diff
@@ -22,13 +22,13 @@
         - cd flux-core
         - ./autogen.sh
         - mkdir build
-        - cd build
         - mkdir install
-        - ../configure --prefix=$(pwd)/install
+        - cd build
+        - ../configure --prefix=${CI_DIRECTORY}/flux-core/install
         - make -j $(nproc)
         ## need to install to get pkgconfig files
         - export CORE_BUILD_DIR=$(pwd)
-        - export CORE_INSTALL_PREFIX=$(pwd)/install
+        - export CORE_INSTALL_PREFIX=${CI_DIRECTORY}/flux-core/install
```

Comment:
Commit message: Actually the issue is that `flux` thinks it is running from the build tree instead of install dir, and will run from the build dir instead of install dir. This makes it so that flux can't find other framework projects which are in the install dir.